mongo/mongo.go

```diff
@@ -114,14 +81,15 @@ func (m *Mongo) GetImages(ctx context.Context, collectionID string) ([]models.Im
 	}
 
 	var results []models.Image
-	err := m.Connection.GetConfiguredCollection().Find(colIDFilter).IterAll(ctx, &results)
+	count, err := m.connection.Collection(m.ActualCollectionName(config.ImagesCollection)).Find(ctx, colIDFilter, &results)
 	if err != nil {
-		if dpMongoDriver.IsErrNoDocumentFound(err) {
-			return nil, errs.ErrImageNotFound
-		}
```

Comment:
Do we need to bring this back?

```
if errors.Is(err, mongodriver.ErrNoDocumentFound) {
	return nil, errs.ErrImageNotFound
}
```

Reply:
No I don't believe this needs to be brought back. Neither the Find (nor the original IterAll()) will return an error when it cannot find a document. The original check:
> `if dpMongoDriver.IsErrNoDocumentFound(err)`
or its updated equivalent:
> `if errors.Is(err, mongodriver.ErrNoDocumentFound)`
will never return true